README.md

```diff
@@ -6,6 +6,31 @@
 
 An [Apache Parquet](https://parquet.apache.org/) implementation in Rust (work in progress)
 
+## Example
+Library usage examples can be found in [src/bin](./src/bin) folder.
+
+### Reading file metadata
```

Comment:
Is it possible we can combine this with `dump-schema`?

Reply:
Yes, we definitely can. I will update README.